src/plugins/envmon/envmon.cpp

```diff
@@ -204,30 +204,30 @@ static event_response_t trap_SspipGetUserName_cb(drakvuf_t drakvuf, drakvuf_trap
     {
         case OUTPUT_CSV:
             csvfmt::print("envmon", drakvuf, info,
-                          keyval("ExtendedNameFormat", ex_name_fmt),
-                          keyval("ExtendedNameFormatStr", std::quoted(ex_name_fmt_str))
+                          keyval("ExtendedNameFormat", fmt::nval(ex_name_fmt)),
+                          keyval("ExtendedNameFormatStr", fmt::qstr(ex_name_fmt_str))
                          );
             break;
         case OUTPUT_KV:
             kvfmt::print("envmon", drakvuf, info,
-                         keyval("ExtendedNameFormat", ex_name_fmt),
-                         keyval("ExtendedNameFormatStr", std::quoted(ex_name_fmt_str))
+                         keyval("ExtendedNameFormat", fmt::nval(ex_name_fmt)),
+                         keyval("ExtendedNameFormatStr", fmt::qstr(ex_name_fmt_str))
                         );
```

Comment:
In this case `OUTPUT_KV` and `OUTPUT_CSV` are mostly the same, what differs is the name of function being called. Maybe we could have something like:

```cpp
case OUTPUT_CSV:
case OUTPUT_KV:
   fmt::print(output_type, "envmon", drakvuf, /* ... */);
```
or `fmt_print<output_type>("envmon", drakvuf, /* ... */);`
and this generic function would route appropriately to `kvfmt::print` or `csvfmt::print` depending on `output_format` value?

Reply:
This way we still can have custom code whenever required but avoid duplication when it's possible.

Reply:
It's only possible then format is known in compile time, but it is runtime info.

Reply:
What about
```cpp
auto printer;
switch (output_type) {
case OUTPUT_KV:
    printer = kvfmt::print;
    break;
    // ... 
}

printer("envmon, drakvuf, info, ...);
```
I'm having trouble thinking of correct type for `printer` tho. I need to write some good C++ code, I'm loosing edge :(

Reply:
Was being added helper func for it.